packages/@o3r/rules-engine/src/components/rules-engine/rule-condition/rule-condition-pres.component.ts

```diff
@@ -35,18 +39,19 @@ export class RuleConditionPresComponent {
    */
   @Input()
   public set condition(condition: TopLevelCondition | undefined) {
+    // eslint-disable-next-line no-underscore-dangle -- false positive
     this._condition = condition;
     this.lhs = (condition as UnaryOperation | BinaryOperation)?.lhs ? this.getOperandName((condition as UnaryOperation | BinaryOperation).lhs) : 'undefined';
     this.rhs = (condition as BinaryOperation)?.rhs ? this.getOperandName((condition as BinaryOperation).rhs) : undefined;
   }
 
   public get condition(): TopLevelCondition | undefined {
+    // eslint-disable-next-line no-underscore-dangle -- false positive
```

Comment:
Same here, the explanation is less about false positive and more the usage of a private property. Isn't there a config to handle this use case? It is weird to allow something on private properties but prevent it when it is on usage (instead of definition)